mixins/test/localize-mixin.test.js

```diff
@@ -297,6 +329,89 @@ describe('LocalizeMixin', () => {
 
 	});
 
+	describe('localizeHTML', async() => {
+
+		const elem = await fixture(`<${localizeHTMLTag}></${localizeHTMLTag}>`);
+
+		const getRenderString = data => {
+			render(data, elem);
+			return elem.innerHTML.replace(/<!--[^]*?-->/g, '');
```

Comment:
It hadn't occurred to me that `render` was something I could import. We still have to attach it to the DOM instead of just getting a string, but I think this is still better than my previous custom function. It improved the output a bit by showing html entities where we would expect rejected/non-templated markup to fail.

Reply:
I haven't tried it myself, but could you use `expect(el).dom.to.equal('...')` from [`@open-wc/semantic-dom-diff`](https://open-wc.org/docs/testing/testing-package/#chai), instead of getting the innerHTML and stripping comments?

Reply:
Works wonderfully!